### PR TITLE
Mention the RabbitMQ management plugin

### PR DESCRIPTION
**Pull request by giacmir**
## Purpose of this pull request

In Magento Cloud RabbitMQ comes with his [management plugin](https://www.rabbitmq.com/management.html) installed. It's a very useful feature not mentioned in documentation. This PR adds a line adding this information with instructions on how to access said feature.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-rabbit.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/cloud/project/project-conf-files_services-rabbit.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ For debugging purposes, it is useful to directly connect to a service instance i
    ```
 
 1. While the session is open, you can start a RabbitMQ client of your choice from your local workstation, configured to connect to the `localhost:<portnumber>` using the port number, username, and password information from the MAGENTO_CLOUD_RELATIONSHIP variable.
+	 The RabbitMQ management plugin is installed in cloud, so there is a graphical utility that can be used for debugging purposes and it is usually available at port 15672.
```
Comment (osrecio) [inline guides/v2.2/cloud/project/project-conf-files_services-rabbit.md]: Hi, 
if usually available at port 15672, but for some reason you're not in this port, how would you know which port you're in? the developer should open a ticket?

Comment (osrecio) [reply guides/v2.2/cloud/project/project-conf-files_services-rabbit.md]: Maybe @hguthrie can talk with Magento Cloud team to clarify it. And you can do the changes addressed from Cloud team.

Comment (hguthrie) [reply guides/v2.2/cloud/project/project-conf-files_services-rabbit.md]: I am researching this. Thanks for the notice! I'll get back to you soon.

Comment (hguthrie) [inline guides/v2.2/cloud/project/project-conf-files_services-rabbit.md]: I'm running into some difficulty finding anyone who uses this. In any case, I would make the following changes:

```suggestion

   To further assist in debugging, you can use the management plugin to access a graphical utility by opening the `http://localhost:15672` URL in your browser.
```

**Comment by shrielenee** [comment]
We are closing this due to inactivity over our 2 week timeframe period. Reopen this PR if you'd like to make the requested changes and have us merge it in! Thanks 🚶‍♀

**Pull request closed without merging**